migrations/0037-add-userAttendancePermission-to-userTable.ts

```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner, TableColumn } from 'typeorm';
+
+const TABLE_NAME = 'Users';
+
+export class AddUserAttendancePermissionToUserTable1691286073346 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.addColumn(TABLE_NAME, new TableColumn({
+      name: 'canSeeAttendance',
+      type: 'boolean',
+      default: true,
```

Comment:
"canSeeAttendance" might be a slightly confusing/overloaded name. Does this mean that this user can't see other peoples' attendances? Or that it can't see its own attendance? Or that other users can't see its own attendance? My understanding is that the latter is what we want, so let's rename this field to something like `isAttendancePublic`